modules/fx/arrow-fx/src/test/kotlin/arrow/fx/IOTest.kt

```diff
@@ -690,19 +690,14 @@ class IOTest : UnitSpec() {
       right.toIO().unsafeRunSync() shouldBe "rightValue"
     }
 
-    "can go from Either to IO by mapping the Left value to a Throwable" {
+    "can go from Either to IO by mapping the Left value to a IO exception" {
 
       val exception = RuntimeException()
-      val left = Either.left("boom")
+      val left = Either.left(exception)
       val right = Either.right("rightValue")
 
-      right
-        .toIOException { exception }
-        .unsafeRunSync() shouldBe "rightValue"
-
-      left
-        .toIOException { exception }
-        .attempt().unsafeRunSync() shouldBe Left(exception)
+      left.toIOException().attempt().unsafeRunSync() shouldBe Left(exception)
```

Comment:
NIT: formatting

Reply:
Probably this should be new line. What are your settings to IDE? after ctrl+alt+l there is no action

Reply:
I think I've added what you meant ;)